packages/kubernetes-1.19/make-kubelet-dirs.conf

```diff
@@ -0,0 +1,5 @@
+[Service]
+# Create the directories for symlinks in /etc
```

Comment:
Both in the commit message and in the systemd services: the directories are created in `/var` not in `/etc`

Reply:
Would this help clarify? The symlinks are still created in `/etc`, and these are the directories they point to.

```suggestion
# Create the backing directories for symlinks in /etc
```

Reply:
Yeah, I think the suggest comment is clearer :+1: